components/hero/HeroLinks.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import { motion } from "framer-motion";
+import { Link } from "@nextui-org/link";
+import Image from "next/image";
+
+import { contactIcon } from "@/public";
+
+import { paragraphVariants3 } from "@/utils";
```

Comment:
Can you update this to be more descriptive? Consider coming back to this project in a year... You want to be able to read your code and understand what paragraphVariants3 is; otherwise, you will have to stop, look it up, and then go back only to continue to repeat the process.